src/config.rs

```diff
@@ -1,9 +1,21 @@
+use log::*;
+use serde::Deserialize;
 use std::error::Error;
 use std::fs;
-use yaml_rust2::{Yaml, YamlLoader};
 
-pub fn load_config(filename: &str) -> Result<Vec<Yaml>, Box<dyn Error>> {
-    let source = fs::read_to_string(filename)?;
-    let conf = YamlLoader::load_from_str(&source)?;
-    Ok(conf)
+#[derive(Deserialize, Default, Debug)]
+pub struct Config {
+    pub konami_code: bool,
+}
+
+pub fn load_config(filename: &str) -> Result<Config, Box<dyn Error>> {
+    let contents = fs::read_to_string(filename)?;
```

Comment:
In regards to logging, maybe we want to add a `warn!` message if we didn't find the file.